e2e/wdio/core/pages/message-strip.po.ts

```diff
@@ -0,0 +1,47 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class MessageStripPo extends CoreBaseComponentPo {
+    private url = '/message-strip';
+    
+    defaultExample = 'fd-message-strip-example ';
+    noIconExample = 'fd-message-strip-noicon-example ';
+    widthExample = 'fd-message-strip-width-example ';
+    playground = '.fd-playground ';
+
+    messageStrip = '.fd-message-strip';
+    messageStripPG = this.playground + this.messageStrip;
+    dismissButton = '.fd-message-strip__close';
+
+    dissmissibleCheckbox = this.playground + 'schema-group div div:nth-child(1) .fd-checkbox'
+    noIconCheckbox = this.playground + 'schema-group div div:nth-child(2) .fd-checkbox'
+    widthInput = this.playground + 'schema-group div div:nth-child(3) input';
+    messageInput = this.playground + 'schema-group div div:nth-child(4) input';
+
+    select = this.playground + 'schema-group select ';
+    warningOption = this.select + '.ng-star-inserted:nth-child(2)';
```

Comment:
please improve selector not to use nth-child
can find option with type = warning then check the messagestrip class is warning too
![2021-07-08_2](https://user-images.githubusercontent.com/47522152/124926509-47905900-e006-11eb-97f5-2aea369dee14.png)